### ISSUE TITLE
Fix contrast for the search field

### ISSUE THREAD

**Issue by jasonnrosa** [HubSpot/cms-theme-boilerplate#282]
When running the boilerplate website through axe we currently get flagged for a lack of contrast on the search field. I believe this may be in reference to the placeholder text.

**Comment by jasonnrosa** [comment]
This is something that will need to be fixed in the default search module. Including no placeholder text doesn't look to remove the placeholder.
